core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1416,6 +1421,11 @@ private void handleErrorInSyncContext(Status error) {
     SubchannelImpl(CreateSubchannelArgs args, LbHelperImpl helper) {
       this.args = checkNotNull(args, "args");
       this.helper = checkNotNull(helper, "helper");
+      subchannelLogId = InternalLogId.allocate("Subchannel", /*details=*/ authority());
+      subchannelTracer = new ChannelTracer(
+          subchannelLogId, maxTraceEvents, timeProvider.currentTimeNanos(),
+          "Subchannel for " + args.getAddresses());
```

Comment:
I would provide full description of the args `"Subchannel for " + args`

Reply:
This PR intends to fix the bug, may not be a good idea to make excessive changes.